sdk/src/Services/S3/Custom/Transfer/_bcl+netstandard/TransferUtility.sync.cs

```diff
@@ -130,57 +148,25 @@ public void Upload(TransferUtilityUploadRequest request)
         /// <inheritdoc/>
         public TransferUtilityUploadResponse UploadWithResponse(string filePath, string bucketName)
         {
-            try
-            {
-                return UploadWithResponseAsync(filePath, bucketName).Result;
-            }
-            catch (AggregateException e)
-            {
-                ExceptionDispatchInfo.Capture(e.InnerException).Throw();
-                return null;
-            }
+            return UploadWithResponseAsync(filePath, bucketName).GetAwaiter().GetResult();
```

Comment:
updating these as per https://github.com/aws/aws-sdk-net/pull/4141#pullrequestreview-3503058236 since i forgot to update them before